docs/glossary.rst

```diff
@@ -0,0 +1,66 @@
+===============================================================================
+XBRL Glossary
```

Comment:
Sphinx has a built-in `glossary::` directive that will be helpful here, and make it easy to reference the defined terms. [See the Sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-glossary)

Reply:
Updated